[] Section14
- [] #Chapter 46 Different Strategies/Options to handle Exceptions in CompletableFuture

- [] #Chapter 47 Handle/Recover Exception using "handle" function

- [] #Chapter 48 Handle/Recover Exception using "handle" function-part2

- [] #Chapter 49 Handle/Recover Exception using "Handle" function-part3

- [] #Chapter 50 Handle/Recover Exception using "exceptionally" function

- [] #Chapter 51 Handle Exception using "whenComplete" function